test/plots/stackedAreaPlotTests.ts

```diff
@@ -2,485 +2,346 @@
+          let areaYs = areaVertices.map((areaVertex) => areaVertex.y).slice(0, -2);
+          data.forEach((datum: any, datumIndex: number) => {
+            let y = linearScale.scale(stackedYs[index][datumIndex]);
+            assert.closeTo(areaYs[datumIndex], y, window.Pixel_CloseTo_Requirement, "cools");
+          });
+        });
+        stackedAreaPlot.destroy();
+        svg.remove();
+      });
     });
 
-    it("pixel positions account for stack offsets", () => {
-      let dataYs = renderer.entities().map((entity) => yScale.invert(entity.position.y));
-      let dataset1Ys = dataset1.data().map((d) => d.y);
-      let dataset2Ys = dataset2.data().map((d, i) => d.y + dataset1.data()[i].y);
-      assert.includeMembers(dataYs, dataset1Ys, "all dataset1 points found");
-      assert.includeMembers(dataYs, dataset2Ys, "all dataset2 points found");
-      svg.remove();
+    describe("setting the y0 property", () => {
+      let stackedAreaPlot: Plottable.Plots.StackedArea<number>;
+
+      beforeEach(() => {
+        let data1 = [
+          {foo: 1},
+          {foo: 3}
+        ];
+        let dataset1 = new Plottable.Dataset(data1);
+        dataset1.metadata({ bar: 7 });
+        let data2 = [
+          {foo: 1},
+          {foo: 3}
+        ];
+        let dataset2 = new Plottable.Dataset(data2);
+        dataset2.metadata({ bar: 10 });
+        stackedAreaPlot = new Plottable.Plots.StackedArea<number>();
+        stackedAreaPlot.addDataset(dataset1);
+        stackedAreaPlot.addDataset(dataset2);
+        stackedAreaPlot.x((d, i) => i);
+        stackedAreaPlot.y((d, i) => i);
+      });
+
+      it("defaults to the 0 constant value", () => {
+        assert.strictEqual(stackedAreaPlot.y0().accessor(null, 0, null), 0, "property set to 0");
+        stackedAreaPlot.destroy();
+      });
+
+      it("can set the y0 property accessor to a constant value", () => {
+        let constantValue = 10;
+        stackedAreaPlot.y0(constantValue);
+        assert.strictEqual(stackedAreaPlot.y0().accessor(null, 0, null), constantValue, "property set to constant value");
+        stackedAreaPlot.destroy();
+      });
+
+      it("can set the y0 property accessor to be dependent on the input data", () => {
+        let accessor = (d: {foo: number}, i: number, ds: Plottable.Dataset) => d.foo * 100 + i * 10 + ds.metadata().bar;
+        stackedAreaPlot.y0(accessor);
+        assert.strictEqual(stackedAreaPlot.y0().accessor, accessor, `property set for datum`);
+        stackedAreaPlot.destroy();
+      });
     });
 
-    it("disables downsampling", () => {
-      assert.strictEqual(renderer.downsamplingEnabled(), false, "downsampling is disabled by default");
-      renderer.downsamplingEnabled(true);
-      assert.strictEqual(renderer.downsamplingEnabled(), false, "downsampling will not be enabled by user");
-      svg.remove();
+    describe("rendering on edge case scenarios", () => {
```

Comment:
Well it's just that `stackedAreaPlot` will operate in a way that we would not recommend users to use.  If someone can come up with a detailed spec on how we want to do this, then we can align to that idea, but right now that spec is very vague